Documentation/Tools/HolographicRemoting.md

```diff
@@ -21,6 +23,34 @@ configure the project using the following steps.
 1. Ensure that **Enable MSBuild for Unity** is selected
 1. Click **Apply**
 
+### Debugging HoloLens 2 remoting via Unity package import
+
+If HoloLens 2 hand joints and eye tracking aren't working over remoting, there are three common points of potential issues. They're listed below in the order you should check them.
+
+#### MSBuildForUnity package import via writing into the package.manifest
+
+For the first point, the best way to check is to open Window -> Package Manager and make sure MSBuild for Unity shows up in the packages list. If it's there, you can assume this step succeeded. If it's not there, try running Mixed Reality Toolkit -> Utilities -> Configure Unity repeat the stops above for running the MRTK Configurator.
```

Comment:
```suggestion
For the first point, the best way to check is to open Window -> Package Manager and make sure MSBuild for Unity shows up in the packages list. If it's there, you can assume this step succeeded. If it's not there, try running Mixed Reality Toolkit -> Utilities -> Configure Unity repeat the steps above for running the MRTK Configurator.
```

Reply:
For steps above, consider adding a link to the steps (because "above" can change as this document gets updated in the future)

Reply:
It's like when you number highway exits A, B, C, and then you want to add another exit between A B and you're out of letters.